code/game/machinery/biogenerator.dm

```diff
@@ -126,66 +158,163 @@ ITEMS
 	name = "Towel"
 	class = BIOGEN_ITEMS
 	object = /obj/item/towel/random
-	cost = 300
+	cost = 200
 
 /singleton/biorecipe/item/jug
 	name = "Empty Jug"
 	object = /obj/item/reagent_containers/glass/fertilizer
-	cost = 100
+	cost = 50
+
+/singleton/biorecipe/item/glowstick
+	name = "Glowstick"
+	object = /obj/random/glowstick
+	cost = 75
+
+/singleton/biorecipe/item/plushie
+	name = "Plushie"
+	object = /obj/random/plushie
+	cost = 200
+
+/singleton/biorecipe/item/yarn
+	name = "Ball of Yarn"
+	object = /obj/random/yarn
+	cost = 75
+
+/singleton/biorecipe/item/knitting
+	name = "Knitting Needles"
+	object = /obj/item/knittingneedles
+	cost = 150
 
 /singleton/biorecipe/item/custom_cigarettes
 	name = "Empty Cigarettes (x6)"
 	object = /obj/item/storage/box/fancy/cigarettes/blank
-	cost = 500
+	cost = 300
 
 /singleton/biorecipe/item/tape_roll
 	name = "Tape Roll"
 	object = /obj/item/tape_roll
-	cost = 250
-
-/singleton/biorecipe/item/botanic_leather
-	name = "Botanical Gloves"
-	object = /obj/item/clothing/gloves/botanic_leather
-	cost = 250
-
-/singleton/biorecipe/item/utility
-	name = "Utility Belt"
-	object = /obj/item/storage/belt/utility
-
-/singleton/biorecipe/item/hydrobelt
-	name = "Hydroponic Belt"
-	object = /obj/item/storage/belt/hydro
-
-/singleton/biorecipe/item/plantbag
-	name = "Plant Bag"
-	object = /obj/item/storage/bag/plants
-	cost = 500
+	cost = 150
 
 /singleton/biorecipe/item/wallet
 	name = "Leather Wallet"
 	object = /obj/item/storage/wallet
 	cost = 100
 
-/singleton/biorecipe/item/satchel
-	name = "Leather Satchel"
-	object = /obj/item/storage/backpack/satchel/leather
-	cost = 400
-
-/singleton/biorecipe/item/cash
-	name = "Money Bag"
-	object = /obj/item/storage/bag/money
-	cost = 400
-
 /singleton/biorecipe/item/soap
 	name = "Soap"
 	object = /obj/item/soap/plant
 	cost = 200
 
+/singleton/biorecipe/item/lighter
+	name = "Cheap Lighter"
+	object = /obj/item/flame/lighter/random
+	cost = 100
+
+/singleton/biorecipe/item/pottedplant
+	name = "Small Plant Pot"
+	object = /obj/random/pottedplant_small
+	cost = 300
+
 /singleton/biorecipe/item/crayon_box
 	name = "Crayon Box"
 	object = /obj/item/storage/box/fancy/crayons
+	cost = 400
+
+/singleton/biorecipe/item/goldstar
+	name = "Box of Gold Stars"
+	object = /obj/item/storage/box/goldstar
 	cost = 600
 
+/singleton/biorecipe/item/googly
+	name = "Box of Googly Eyes"
+	object = /obj/item/storage/box/googly
+	cost = 600
```

Comment:
please no
I don't want to see people spamming googly eyes on everything every round